.travis.yml

```diff
@@ -1,24 +1,48 @@
 os:
-  - linux
-  - osx
-
-# This makes us use Ubuntu 14 instead of 12
+- linux
+- osx
 dist: trusty
-
-# Notes:
-# - Travis uses cmake 3.0.2 on OS X; we need 3.1 or newer (thanks tbodt)
-
 language: c
 script:
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo apt-get update; fi
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo apt-get install libgtk-3-dev -y || sudo apt-cache search libgtk3; fi
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew update; fi
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew upgrade cmake; fi 
-  - mkdir build
-  - cd build
-  - cmake --version
-  - cmake .. -G "Unix Makefiles"
-  - make tester examples
-  - rm -rf *
-  - cmake .. -G "Unix Makefiles" -DBUILD_SHARED_LIBS=OFF
-  - make tester examples
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo apt-get update; fi
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo apt-get install libgtk-3-dev -y ||
+  sudo apt-cache search libgtk3; fi
+- if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew update; fi
+- if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew upgrade cmake; fi
+- mkdir build
+- mkdir artifacts
+- cd build
+- cmake --version
+- cmake .. -G "Unix Makefiles" -DBUILD_SHARED_LIBS=OFF
+- make tester examples
+- cd out
+- tar -cvf ../../artifacts/examples-static-$TRAVIS_OS_NAME-$TRAVIS_TAG.tar.gz controlgallery controlgallery histogram
+- tar -cvf ../../artifacts/libui-static-$TRAVIS_OS_NAME-$TRAVIS_TAG.tar.gz libui.a
+- cd ..
+- rm -rf *
+- cmake .. -G "Unix Makefiles"
+- make tester examples
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then rm out/libui.so; fi
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then mv out/libui.so.0 out/libui.so; fi
```

Comment:
This is incorrect; the installed name of a so on linux will always have the version suffix, with the versionless name being just a symlink